DataLayer/Wearable/build.gradle

```diff
@@ -92,4 +93,15 @@ dependencies {
     implementation libs.wear.compose.foundation
 
     implementation libs.playservices.wearable
+
+  // Horologist for correct Compose layout
+  implementation libs.horologist.composables
+  implementation libs.horologist.compose.layout
+  implementation libs.horologist.compose.material
+
+  coreLibraryDesugaring libs.desugar.jdk.libs
+
+  // Preview Tooling
+  implementation libs.compose.ui.tooling.preview
```

Comment:
I think, but always get this wrong.  This should be the debugImplementation. And tooling without preview should be implementation.